safe_web_api_playground/static/js/code-snippets/safeApp.js

```diff
@@ -0,0 +1,152 @@
+module.exports = {
+  safeApp: {
+
+    initialise: () => {
+
+      // Welcome! Get started by running this snippet.
+      // Then open the `safeApp` module on the left-hand side of the screen.
+      // Reference documentation: http://docs.maidsafe.net/beaker-plugin-safe-app/
+      // When not focused in this text box, simply use your `Enter` key at any time to run code snippets.
+
+      let appInfo = {
+        id: 'net.maidsafe.api_playground.webclient.10',
+        name: 'SAFE browser API playground',
+        vendor: 'MaidSafe Ltd.'
+      };
+
+      return window.safeApp.initialise(appInfo)
+      .then(function(res) {
+        appToken = res;
+        return 'Returns app token: ' + res;
```

Comment:
We are changing the name in the API from `appToken` to `appHandle` since it's more consistent and correct with what it actually is. Can you please rename them here in the sample snippets too?